examples/advanced/ads.js

```diff
@@ -29,7 +30,13 @@ var Ads = function() {
   // Start ads when the video player is clicked, but only the first time it's
   // clicked.
   this.clickedOnce = false;
```

Comment:
Why not remove the event listener once the click/tap happens?

You can use something like
```e.target.removeEventListener(e.type, arguments.callee);``` within the function to remove itself.